torchci/clickhouse_queries/lf_rollover_percentage/params.json

```diff
@@ -0,0 +1,4 @@
+{
+  "days_ago": "Int64",
+  "granularity": "String"
```

Comment:
I never actually see granularity used?  Also curious as to if you want to use the timeParams instead of days_ago but its up to you

Reply:
true, bad copy/pasta there on granularity. Removed now.

Re: days_ago...I might do it in a future PR